content/docs/guides/requirements.md

```diff
@@ -11,17 +11,14 @@ aliases = ["/docs/guides/"]
 +++
 
 * [Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
-* [Tensorflow](https://www.tensorflow.org/get_started/)
-* [Ksonnet](https://ksonnet.io/docs/tutorial)
+* [TensorFlow](https://www.tensorflow.org/get_started/)
+* [ksonnet](https://ksonnet.io/docs/tutorial)
 
 For an end to end example illustrating in details how to deploy kubeflow and run a training job from scratch, check out this [tutorial](https://ai.intel.com/lets-flow-within-kubeflow/).
 
-## Requirements
- * Kubernetes >= 1.8 [see here](https://github.com/kubeflow/tf-operator#requirements)
- * ksonnet version [0.11.0](https://ksonnet.io/#get-started). (See [below](#why-kubeflow-uses-ksonnet) for an explanation of why we use ksonnet)
- * An existing kubernetes cluster:
+ * ksonnet version {{% ksonnet-min-version %}} or later. See the [ksonnet component page](/docs/guides/components/ksonnet/) for an explanation of why we use ksonnet.
```

Comment:
Similar to my previous comment, this partial merge of content from the master branch doesn't work well here.